src/app/components/CourseList/CourseList.tsx

```diff
@@ -0,0 +1,84 @@
+"use client";
+
+import React, { useState } from 'react';
+import { FaChevronDown } from 'react-icons/fa';
+import courses, { Course, categoriesList, ToolCategory } from '../../../../src/CourseListData/CourseListData';
+
+const CategoryList: React.FC = () => {
+  const [categoryOpen, setCategoryOpen] = useState(false);
+  const [developmentOpen, setDevelopmentOpen] = useState(false);
+  const [toolsOpen, setToolsOpen] = useState(false);
+
+  const toggleCategory = () => {
+    setCategoryOpen(!categoryOpen);
+  };
+
+  const toggleDevelopment = () => {
+    setDevelopmentOpen(!developmentOpen);
+  };
+
+  const toggleTools = () => {
+    setToolsOpen(!toolsOpen);
+  };
+
+  const handleDevelopmentChange = (optionIndex: number) => {};
+  const handleToolChange = (optionIndex: number) => {};
+
+  return (
+    <div>
+      <div style={{ display: 'flex', alignItems: 'center' }} onClick={toggleCategory}>
+        <h2 style={{ cursor: 'pointer' }}>Category</h2>
+        <FaChevronDown style={{ marginLeft: '5px', cursor: 'pointer' }} />
+      </div>
+      {categoryOpen && (
+        <ul>
+          {categoriesList.map((category, index) => (
+            <React.Fragment key={index}>
```

Comment:
React.Fragment birden fazla öğeyi sarmak için kullanılır. Buradaz zaten ul kullanılacağı için üst taraftaki ul'u React.Fragment yerine kullanabilirsin